libs/core/src/components/card/card.ts

```diff
@@ -69,9 +71,8 @@ export class GdsCard extends GdsContainer {
     super()
 
     // Default values for cards
-    this.padding = 's; m{l}'
-    this['border-radius'] = 'xs; m{s}'
-    this['border-width'] = '4xs'
+    this.padding = 's;m{l}'
+    this['border-radius'] = 'xs;m{s}'
```

Comment:
I would suggest here that we add as default display to `flex` reason for that is most times we are having to wrap again content to do that, most common is flex-direction column. 

```ts
this.display = 'flex'
this['flex-direction'] = 'column'
this['gap'] = 's;m{l}'
```

Reply:
Good catch, I agree that flex makes good sense as a default for card!